tests/cecil-tests/AttributeTest.cs

```diff
@@ -37,7 +37,7 @@ public class AttributeTest {
 		//
 		// This test should find Extension, note that it has an ios attribute,
 		// and insist that some maccatalyst must also be set.
-		// [TestCaseSource (typeof (Helper), "NetPlatformAssemblies")]
+		[TestCaseSource (typeof (Helper), "NetPlatformAssemblies")]
```

Comment:
This avoids a string constant:

```suggestion
		[TestCaseSource (typeof (Helper), nameof (Helper.NetPlatformAssemblies))]
```